src/routes.js

```diff
@@ -660,6 +659,41 @@ const routes = [
     }
   }),
 
+  asyncRoute({
+    path: '/projects/:projectId([1-9]\\d*)/forms/:xmlFormId/submissions/:instanceId/edit',
+    component: 'FormSubmission',
+    name: 'WebFormEditSubmission',
+    props: true,
+    loading: 'page',
+    meta: {
+      standalone: true,
+      title: () => [form.nameOrId],
+    }
+  }),
+  asyncRoute({
+    path: '/projects/:projectId([1-9]\\d*)/forms/:xmlFormId/submissions/new',
+    component: 'FormSubmission',
+    name: 'WebFormNewSubmission',
+    props: true,
+    loading: 'page',
+    meta: {
+      standalone: true,
+      title: () => [form.nameOrId],
+    }
```

Comment:
I'm noticing that we're not using `validateData` to check that the user is allowed to `submission.create`. I guess we don't even fetch the project on this page. Is the plan just to have Backend validate that aspect of things, returning a 403 if the user (e.g., a project viewer) tries to create a submission when they aren't allowed to?